WcaOnRails/app/controllers/api/v0/roles_controller.rb

```diff
@@ -41,6 +41,17 @@ class Api::V0::RolesController < Api::V0::ApiController
       end
     end
 
+    unless is_active.nil?
```

Comment:
Nit: Rails has `if is_active.present?` which I think is a tad bit more readable because there's no negation baked into the `unless` condition

Reply:
Also valid for the `unless status.nil?` --> `if status.present?` above